pkgs/applications/misc/termdown/default.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs, stdenv, fetchFromGitHub, buildPythonApplication,
+click, pyfiglet, dateutil}:
+
+with stdenv.lib;
+
+buildPythonApplication rec {
+
+  name    = "termdown-${version}";
+  version = "1.11.0";
+
+  src = pkgs.fetchFromGitHub {
+      rev    = "d1e3504e02ad49013595112cb03fbf175822e58d";
+      sha256 = "1i6fxymg52q95n0cbm4imdxh6yvpj3q57yf7w9z5d9pr35cf1iq5";
+      repo   = "termdown";
+      owner  = "trehn";
+  };
+
+  propagatedBuildInputs = [ dateutil click pyfiglet ];
+
+  meta = {
```

Comment:
You can do meta = with stdenv.lib; {

That way you don't have to write stdenv.lib in licenses and platforms